CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
-## [Unreleased]
+## [5.1.0]
+
+### Added
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Extend JSON reader functionality
+- Support loading schema from JSON data
+- Configure decimal places in JSON
+- Export valid Shapefiles for mappings, including feature types with no geometry
+- Support downloading WFS files to the tmp folder for bucket service
+- Feature: Type selection broken on custom export config
+- Support adding multiple selections of feature types to be exported to CSV
+- Replace default map with OpenStreetMap
+- Add Boolean and Java Pattern class to Groovy restriction whitelist
+- Update decimal places to 20 for GeoTools
+
+## Maintenance
+- Update Geotools to version 29.1
+- Update dependencies in hale-platform to resolve security issues
+- Fix for the Windows installer
+ 
+### Fixed
+- Fix Excel reader not adding instances for empty rows
+- Fix project settings, warnings, include JSON UI extension definition in the build
+- Fix Shapefile writer to recognize geometries for feature types
+- Fix exporting transformed data leading to a shift of attribute values in Shapefile
+- Fix TopoJSON encoding
+- Fix endless loop when skipping instances in paginated WFS request
+- Fix exporting HSD schemas for schemas with no namespace
+
 
 ## [5.0.1]
```

Comment:
The link here would be `https://github.com/halestudio/hale/compare/5.0.0...5.0.1`